db/commitlog/replay_position.hh

```diff
@@ -91,9 +91,6 @@ private:
     replay_position _rp;
 };
 
-
-std::ostream& operator<<(std::ostream& out, const replay_position& s);
-
```

Comment:
Why drop this when it is apparently needed in tests? Why not just put the fmt-based ostream op in scylla-lib? Unless used it should drop in linking, no?

Reply:
> Why drop this when it is apparently needed in tests?

@elcallio hi Calle, actually i want to make it difficult to use `operator<<`. and make people switch over to {fmt}. two parallel formatting systems are too much for us. both because they are difficult to maintain, also because in some cases, it's wrong. 

> Why not just put the fmt-based ostream op in scylla-lib? Unless used it should drop in linking, no?

are you proposing add a library named `scylla-lib` ?

Reply:
Having scylla as a .a and link tests/exec that way has been discussed at end for ages. But same principle applies to .o linkage so moot.

Fair enough about ostream ops.